config/locales/de.label.yml

```diff
@@ -124,4 +124,4 @@ de:
       :title: Dresden.rb
       :subtitle: Ruby User Group Dresden
       :meta_desc: Die neue Ruby User Group in Dresden
-      :custom_recurrence: Erster Termin am 28.11.2024 
+      :custom_recurrence: aller drei Monate
```

Comment:
```suggestion
      :custom_recurrence: alle drei Monate
```

Reply:
Thank you!

Reply:
I would love better commit messages. ;)

Reply:
Do you prefer a commit for this fixup with a better message vs one commit for the whole PR with the fix included? I'd normally squash this but I can also write a small commit message for this fixup.

Reply:
There is nothing to squash when there is only one commit/change and no one benefits from fix commits within a branch that introduces the change. 

To be honest, I would have amended the typo fix without even asking.

Reply:
Yeah I was just lazy and used the "Commit suggestion" button from Github and thought it'd be easy to squash with Github's squash merge. I did the rebase manually now.